src/main/java/gameboy/gamepack/racinggame/data/entity/Driver.java

```diff
@@ -0,0 +1,35 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import java.util.Random;
+
+public class Driver {
+
+    private final Random random;
+    private static final int MOVE_RANGE = 10;
+    private static final int MOVE_MIN = 4;
+    private Car car;
+
+    public Driver() {
+        this.random = new Random();
+        this.car = new Car();
+    }
+
+    public Driver(Random random) {
+        this.random = random;
+        this.car = new Car();
+    }
```

Comment:
```suggestion
    public Driver() {
        this(new Random());
    }

    public Driver(Random random) {
        this.random = random;
        this.car = new Car();
    }
```

주 생성자 하나를 사용하는 것이 좋습니다.
초기화 로직을 여러번 작성할 필요가 없으니까요 😄